src/test/java/uk/gov/hmcts/reform/bailcaseapi/domain/entities/ccd/StateTest.java

```diff
@@ -14,14 +14,13 @@ void has_correct_values() {
         assertEquals("applicationEnded", State.APPLICATION_ENDED.toString());
         assertEquals("applicationSubmitted", State.APPLICATION_SUBMITTED.toString());
         assertEquals("bailSummaryUploaded", State.BAIL_SUMMARY_UPLOADED.toString());
-        assertEquals("uploadBailSummary", Event.UPLOAD_BAIL_SUMMARY.toString());
-        assertEquals("recordTheDecision", Event.RECORD_THE_DECISION.toString());
```

Comment:
These two lines are wrong because they're asserting that these values from the `Event` enum equal these strings, but the test is for State.java.
It does miss the check for `State.RECORD_THE_DECISION` though, so I'll add that in.